# This message is to control the behaviour of the line follow controller

# Which edge to follow
bool RIGHT = true
bool LEFT = false
bool follow false

float32 offset